api/Controllers/OrdersController.cs

```diff
@@ -0,0 +1,141 @@
+﻿using AutoMapper;
+using AutoMapper.QueryableExtensions;
+using iTechArt.Hotels.Api.Entities;
+using iTechArt.Hotels.Api.Models;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using static iTechArt.Hotels.Api.Constants;
+
+namespace iTechArt.Hotels.Api.Controllers
+{
+    [Route("api")]
+    [ApiController]
+    public class OrdersController : Controller
+    {
+        private readonly HotelsDatabaseContext _hotelsDb;
+        private readonly IMapper _mapper;
+
+        public OrdersController(HotelsDatabaseContext hotelDb, IMapper mapper)
+        {
+            _hotelsDb = hotelDb;
+            _mapper = mapper;
+        }
+
+        [Route("orders")]
+        [HttpPost]
+        [Authorize(Roles = Role.Client)]
+        public async Task<IActionResult> AddOrder([FromBody] OrderToAdd order)
```

Comment:
Why do you need to send facilities that were not marked as selected

Reply:
It Is better to do in in frontend application, I am agree with you